spec.html

```diff
@@ -32347,7 +32497,7 @@ <h1>Date Time String Format</h1>
                 `Z`
               </td>
               <td>
-                is the UTC offset representation specified as *"Z"* (for UTC with no offset) or an offset of either *"+"* or *"-"* followed by a time expression `HH:mm` (indicating local time ahead of or behind UTC, respectively)
+                is the UTC offset representation specified as *"Z"* (for UTC with no offset) or an offset of either *"+"* or *"-"* followed by a time expression `HH:mm` (indicating local time ahead of or behind UTC, respectively, which is a subset of the <emu-xref href="#sec-time-zone-offset-strings">time zone offset string format</emu-xref>.)
```

Comment:
What is the benefit of this addition?

Reply:
Since we have this ad-hoc format here as well as the newly added time zone offset string format below, I thought it'd be good to explain what the relationship was between them.

Reply:
Hmm, I see. Perhaps the relationship can be expressed more strongly.
```suggestion
                is the UTC offset representation specified as *"Z"* (for UTC with no offset), or as either *"+"* or *"-"* followed by a time expression `HH:mm` (a subset of the <emu-xref href="#sec-time-zone-offset-strings">time zone offset string format</emu-xref> for indicating local time ahead of or behind UTC, respectively)
```